lib/windows.cc

```diff
@@ -258,11 +258,19 @@ Napi::Boolean showWindow (const Napi::CallbackInfo& info) {
 
 Napi::Boolean bringWindowToTop (const Napi::CallbackInfo& info) {
     Napi::Env env{ info.Env () };
-
     auto handle{ getValueFromCallbackData<HWND> (info, 0) };
     BOOL b{ SetForegroundWindow (handle) };
-    SetActiveWindow (handle);
-    SetWindowPos (handle, HWND_TOP, 0, 0, 0, 0, SWP_SHOWWINDOW | SWP_NOMOVE | SWP_NOSIZE);
+
+    HWND hCurWnd = ::GetForegroundWindow();
+    DWORD dwMyID = ::GetCurrentThreadId();
+    DWORD dwCurID = ::GetWindowThreadProcessId(hCurWnd, NULL);
+    ::AttachThreadInput(dwCurID, dwMyID, TRUE);
+    ::SetWindowPos(handle, HWND_TOPMOST, 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE);
+    ::SetWindowPos(handle, HWND_NOTOPMOST, 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE);
+    ::SetForegroundWindow(handle);
```

Comment:
This line returns a boolean which needs to be assigned to b.

Reply:
Sorry what line are you referring to?

Reply:
`::SetForegroundWindow(handle);`